modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/resources/META-INF/resources/js/SystemDataSets.tsx

```diff
@@ -21,20 +27,121 @@ import openDefaultFailureToast from './utils/openDefaultFailureToast';
 import openDefaultSuccessToast from './utils/openDefaultSuccessToast';
 import {IDataSet, ISystemDataSet} from './utils/types';
 
+interface IFrontendDataSetContext {
+	onSelect: Function;
+	selectItems: Function;
+	selectable: boolean;
+	selectedItemsKey: keyof ISystemDataSet;
+	selectedItemsValue: Array<any>;
+}
+
+const SystemDataSetModalList = ({
```

Comment:
minor nit on naming, which I oversaught on the previous review. The "modal" part looks unnecesary as this is just a FDS view component. This looks like `SystemDataSetListView` or `SystemDataSetsView`

Reply:
Ah good point, I'll update the naming.